src/js/components/ThumbsRating/ThumbsRating.js

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Like, LikeFill, Dislike, DislikeFill } from 'grommet-icons';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+const ThumbsRating = ({ ...rest }) => {
+  const theme = useContext(ThemeContext);
+  return (
+    <RadioButtonGroup direction="row" options={['like', 'dislike']} {...rest}>
+      {(option, { checked }) => {
+        if (option === 'like') {
+          return checked ? (
+            <LikeFill
+              color={
+                theme.thumbsRating?.like?.icon?.color ||
+                theme.thumbsRating?.like?.icon?.color?.fill
+              }
+            />
+          ) : (
+            <Like
+              color={
+                theme.thumbsRating?.like?.icon?.color ||
+                theme.thumbsRating?.like?.icon?.color?.outline
+              }
+            />
+          );
+        }
+        return checked ? (
+          <DislikeFill
+            color={
+              theme.thumbsRating?.dislike?.icon?.color ||
```

Comment:
feel free to add ideas for naming in theme.

Reply:
I'm fine with the naming that is already here. Maybe we could shorten it to `thumbsRating.dislike.color` because there is nothing else I would expect to be able to change the color on here besides the icon, so icon is already implied. Soft opinion though, flexible either way.